plugins/modules/events_and_notifications_workflow_manager.py

```diff
@@ -2304,7 +2279,14 @@ def get_diff_merged(self, config):
                 self.log(self.msg, "ERROR")
                 return self
 
-            if not port.isdigit() or (isinstance(port, int) and port not in range(1, 65536)):
+            if isinstance(port, str):
+                if not port.isdigit() or (int(port) not in range(1, 65536)):
+                    self.status = "failed"
+                    self.msg = "Invalid Syslog destination port '{0}' given in playbook. Select port from the number range(1, 65535)".format(port)
```

Comment:
Please choose a port within the range of numbers (1, 65535)

Reply:
updated.